src/interaction/orbitalnavigator.cpp

```diff
@@ -232,6 +237,7 @@ OrbitalNavigator::OrbitalNavigator()
     , _velocitySensitivity(VelocityZoomControlInfo, 0.02f, 0.01f, 0.15f)
     , _applyLinearFlight(ApplyLinearFlightInfo, false)
     , _flightDestinationDistance(FlightDestinationDistanceInfo, 2e8f, 0.0f, 1e10f)
+    , _flightDestinationFactor(FlightDestinationFactorInfo, 1E-4)
```

Comment:
May consider also adding a max and min value for the GUI property.

Reply:
Got it. Updated!